common/notification/actions.ts

```diff
@@ -38,6 +38,9 @@ const _notificationActions = {
     pupil_screening_add: {
         description: 'Pupil / Screening was added',
     },
+    pupil_screening_update: {
```

Comment:
Maybe two actions instead, one for success and one for rejection? Then one can also add two different notifications ...

Reply:
Okay 👍 
I have no idea how that system works